rst_files/julia_essentials.rst

```diff
@@ -851,38 +861,43 @@ Another place that you may use a ``Ref`` is to fix a function parameter you do n
 
 .. code-block:: julia
 
-    f(x, y) = [1, 2, 3] ⋅ x + y # "⋅" can be typed by \cdot<tab>
-    f([3, 4, 5], 2) # uses vector as first parameter
-    f.(Ref([3,4, 5]), [2,3]) # broadcasting over 2nd parameter, fixing first
+    f(x, y) = [1, 2, 3] ⋅ x + y   # "⋅" can be typed by \cdot<tab>
+    f([3, 4, 5], 2)               # Uses vector as first parameter
+    f.(Ref([3, 4, 5]), [2, 3])    # Broadcasting over 2nd parameter, fixing first
 
 
 Scoping and Closures
 =====================
 
 Since global variables are usually a bad idea, we will concentrate on understanding the role of good local scoping practice
 
-That said, many of the variables in a Jupyter notebook are global, even the whole text could be copied into a function
+That said, global variables are often convenient and easy to use in Jupyter notebooks
```

Comment:
This is missing the spirit of the comment (which is: you can copy the whole thing inside of a function and they cease to be global).  But the comment is very unclear.  How about something like this instead (which you could edit and merge)
```
That said, while many of the variables in these Jupyter notebook are global, we have been careful to write the code so that the entire code could be copied inside of a function

When copied inside a function, variables become local and functions become closures
```